doc/internal/man3/ossl_provider_new.pod

```diff
@@ -148,6 +150,9 @@ be located in that module, and called.
 
 =back
 
+ossl_provider_available() checks if given provider object I<prov> is
+"activated" or a fallback.
```

Comment:
This could do with tweaking - if fallbacks are not yet activated it will activate them (if appropriate), and *then* check if the provider is activated.

Reply:
Done